assets/src/edit-story/components/canvas/useInsertElement.js

```diff
@@ -74,6 +74,15 @@ function useInsertElement() {
       if (resource) {
         backfillResource(resource, elementId);
       }
+      // Auto-play on insert.
+      if (type === 'video') {
+        setTimeout(() => {
+          const videoEl = document.getElementById(`video-${elementId}`);
+          if (videoEl) {
+            videoEl.play();
+          }
```

Comment:
`else` try again in a few millis? Or just ignore it?

Reply:
I tested that with 6x throttled CPU and a lot of elements, it was still fine.
If anyone finds a bug here - we can try to play it instantly (0ms) and at the longest time considered responsive (100ms), double .play() will not hurt us.